src/workflows-app/SubmissionConfig.js

```diff
@@ -197,6 +201,14 @@ export const BaseSubmissionConfig = (
         message: 'You may check on the progress of workflow on this page anytime.',
         timeout: 5000,
       });
+      captureEvent(Events.workflowsAppLaunchWorkflow, {
+        ...extractWorkspaceDetails(workspace),
+        methodName: method.name,
+        methodId,
+        methodVersionId: selectedMethodVersion.method_version_id,
+        methodSource: method.source,
+        previouslyRun: method.last_run.previously_run,
+      });
```

Comment:
This function will ignore errors, meaning the user flow will still be preserved even if the metrics logging fails for some reason

Reply:
I may be misunderstanding where method IDs come from. My understanding was that method IDs are shared across workspaces for the same method, for example `fetch_sra_to_bam` would have the same ID regardless of workspace? My thought is that this would help us identify unique workflows in case we end up with two or more workflows named `fetch_sra_to_bam`? I am not familiar with how methods are stored/tracked, so would appreciate some clarity there.

Reply:
Ah, but each workspace has their own CBAS database, so they wouldn't necessarily agree upon IDs. Thank you for pointing this out, I will take a look and see what may be more useful (such as source url)